src/fat.ts

```diff
@@ -0,0 +1,86 @@
+import { FileHandle, open } from 'node:fs/promises';
+import { MachoFile } from './macho';
+
+const fatMagic = [0xca, 0xfe, 0xba, 0xbe];
+const sizeOfFatMagic = fatMagic.length
+const lengthOfCpuType = 4;
+const lengthOfCpuSubType = 4;
+const lengthOfMachoStartOffset = 4;
+const lengthOfMachoSize = 4;
+const lengthOfAlign = 4;
```

Comment:
I wonder if these constants are defined in some sort of blessed open source file for fat format (rather than defining it yourself). Just curious, no need to actually do that. The fat file format won't be changing anytime soon.

Reply:
Unfortunately I don't see any FAT or Universal Binary parsers on npm 😩.